src/localize.ts

```diff
@@ -12,26 +12,37 @@ interface PartialI18nDictionary {
 
 type RootI18nDictionary = Record<string, PartialI18nDictionary>;
 
+type Container = {
+  [container: string]: PartialI18nDictionary;
+};
+
+type Value = {
+  [value: string]: Record<string, any>;
+};
```

Comment:
This type isn't correct, you see the `interpolate` function accepts a flat object rather than nested records.

Reply:
Hi @logaretm,

Thanks for your feedback, I found where I did wrong.

Now, since `interpolate` is expecting `values` to be `Record<string, any>` I did the same here for `values`.

Also I used the same `Value` type to keep logic separated from functions instead of adding it directly to the parameter.

Please let me know if something is not right yet.